tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/CharInfo.java

```diff
@@ -53,7 +53,7 @@ public EscapedCharEntry(final CodePointEscaper c) {
 
     public static final class EscapedCharInfo {
         public final String forceEscapeRegex =
-                CodePointEscaper.ESCAPE_IN_SURVEYTOOL.toPattern(true);
+                CodePointEscaper.regexPattern(CodePointEscaper.ESCAPE_IN_SURVEYTOOL, "\\u{", "}");
```

Comment:
@macchiati i'm assuming this is how you expected this to be used

Reply:
yes. As it turns out, there are only really two choices, \u style or \x style, so the API can be simplified, but TBD for now.